src/PwrDrvr.LambdaDispatch.Router/LambdaConnection.cs

```diff
@@ -324,8 +327,8 @@ private async Task RelayResponseFromLambda(HttpResponse incomingResponse)
       // Read until we fill the bufer OR we get an EOF
       int totalBytesRead = 0;
       int idxToExamine = 0;
-      int idxPriorLineFeed = -1;
-      int idxHeadersLast = -1;
+      int idxPriorLineFeed = int.MinValue;
```

Comment:
This would match the condition below if the first char was `\n` since `i` would be `0` in that case and `i-1 = -1` which was equal to what this was previously initialized to.